dot/network/service.go

```diff
@@ -368,6 +376,21 @@ func (s *Service) updateMetrics() {
 	}
 }
 
+func getProcessStartTime() float64 {
```

Comment:
Noob question, do we actually need the exact process start time? Can't we just return the `time.Now().Unix()`? 🤔 Asking just in case, since os/process stuff sometimes is flakky from one os platform to another.

Reply:
I'm not sure, zombienet uses this prometheus metric `substrate_process_start_time_seconds` to confirm node has started.  So, I don't think this needs to be exact, so perhaps I should use `time.Now.Unix()`